manifests/charts/global.yaml

```diff
@@ -57,6 +57,10 @@ global:
   # One central istiod controls all remote clusters: disabled by default
   centralIstiod: false
 
+  # If true, listener filter original_src will be added for inbound listeners of sidecars
+  # running in TPROXY interception mode
+  tproxyPreserveSourceIP: false
```

Comment:
We do not need to add a new option. Users can set it directly with `pilot.env.TPROXY_PRESERVE_SOURCE_IP=true`

Reply:
OK, this commit reverted.